src/GeositeFramework/plugins/measure/AgsMeasure.js

```diff
@@ -353,13 +359,13 @@ define(["jquery",
                 
                 // If the user has drawn the polygon ring anti-clockwise, reverse the ring
                 // to make it a valid esri geometry.
-                if (!Geometry.isClockwise(_points)) {
-                    _points = _points.reverse();
-                }
+                //if (!Polygon.isClockwise(_points)) {

```

Comment:
Looks like this method has been moved from the class to the object instance, and without it it's producing negative areas for counter-clockwise polygons.  The if statement could be brought back with something like: 

```javascript
if (!polygon.isClockwise(_points) {
    _points = _points.reverse();
}
```